sys-devel/icecream/files/icecream.openrc

```diff
@@ -0,0 +1,55 @@
+#!/sbin/openrc-run
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+depend() {
+	need net
```

Comment:
Is this necessary?

Reply:
I don't know.  This is a rework of the older script, that is a relic.  In my limited testing, it does start with only the loopback interface, but I'm not sure how it would behave with zero interfaces.  Both `iceccd` and `icecc-scheduler` are network daemons and `need net` seems to be pretty standard from what I've seen; sys-devel/distcc uses this pattern, as does Nginx.